src/string_util.cc

```diff
@@ -29,6 +29,17 @@ const string_view default_whitespace_characters{ " \t\r\n\a\b\f\v" };
 const std::array<char, 16> hex_digits{
     { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'} };
 
+std::string repeat(gul14::string_view str, std::size_t n)
+{
+    std::string result;
+
```

Comment:
Maybe a small optimization, usually Lars' job to suggest these ;-)

```suggestion
    if (n == 0 or str.empty())
        return "";
```

Reply:
Hm... that raises the number of lines of x86-64 assembly from 54 to 82 on Godbolt (gcc 12.1), and if I understand correctly it disables NRVO. Does it improve performance for these corner cases? Maybe, but we'd need benchmarks to prove that. I would rather leave it out.

Reply:
I'm fine with that.

I already approved.